test/foundry/access/AccessController.t.sol

```diff
@@ -1372,12 +1372,14 @@ contract AccessControllerTest is BaseTest {
         );
         vm.prank(owner);
         mockNFT.burn(tokenId);
-        vm.expectRevert(abi.encodeWithSelector(ERC721NonexistentToken.selector, tokenId));
-        accessController.getPermission(
-            address(ipAccount),
-            signer,
-            address(mockModule),
-            mockModule.executeSuccessfully.selector
+        assertEq(
```

Comment:
Why is the expectRevert removed & replaced with assertEq? Behavior change?

Reply:
correct, the behavior is changing as migrate to solady, the soladay 6551 owner() doesn't revert when bound NFT was burnt, it return zero address instead.